src/containers/App/reducer.js

```diff
@@ -0,0 +1,13 @@
+const initialState = {
+  buildInformation: {
+    frontEndBuldNumber: process.env.REACT_APP_BUILD_NUMBER,
+    deploymentEnvironment: process.env.REACT_APP_DEPLOYMENT_ENV,
+  },
+};
+
+export default (state = initialState, action) => {
```

Comment:
reducers should be named since you can multiple ones per file. Plus destructuring is really important since its easier to read what has been access by the object. That is the reason for the: `const { type } = action`.
In this way if another dev sends another prop to action they just add this to this line which is at the beginning of the function and its pretty clear what is inside action and what has been accessed.

```javascript
const app = (state = initialState, action) =>  {
  const { type } = action;
  switch (type) {
    default:
      return state;
  }
}

export default app;
```